### PR TITLE
docs: add new how-to guide

### PR DESCRIPTION
**Pull request by cjdcordeiro**
This guide aims at converting a popular entrypoint from and existing Docker image (nginx) into a Pebble layer that can be used from within a ROCK.

- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

File: docs/how-to/code/convert-to-pebble-layer/task.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+execute: |
+  # [docs:pack]
+  rockcraft
+  # [docs:pack-end]
+
+  # [docs:skopeo]
+  skopeo copy oci-archive:custom-nginx-rock_latest_amd64.rock docker-daemon:custom-nginx-rock:latest
+  # [docs:skopeo-end]
+
```
Comment (tigarmo) [inline docs/how-to/code/convert-to-pebble-layer/task.yaml:18]: please remove the .rock file here

Comment (cjdcordeiro) [reply docs/how-to/code/convert-to-pebble-layer/task.yaml:18]: done

File: docs/how-to/convert-to-pebble-layer.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Design the Pebble services
+--------------------------
+
+A `Pebble layer
+<https://github.com/canonical/pebble/#layer-specification>`_
+is composed of metadata, checks and services. The latter is present in
+``rockcraft.yaml`` as a `top-level field
+<https://canonical-rockcraft.readthedocs-hosted.com/en/latest/
+reference.html#format-specification>`_
+and is represents the services which are loaded by the Pebble entrypoint when
```
Comment (cjdcordeiro) [inline docs/how-to/convert-to-pebble-layer.rst]:
```suggestion
and it represents the services which are loaded by the Pebble entrypoint when
```

**Review by tigarmo — APPROVED** [review]
Thanks!

File: docs/how-to/code/convert-to-pebble-layer/task.yaml [hunk 3]
```diff
@@ -24,4 +26,5 @@ execute: |
   curl localhost:8080
   # [docs:curl-end]
 
-  docker rm -f nginx-pebble-service
\ No newline at end of file
+  docker rm -f nginx-pebble-service
+  docker rmi -f custom-nginx-rock:latest
```
[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply docs/how-to/code/convert-to-pebble-layer/task.yaml:30]: noted

**Review by mr-cal — APPROVED** [review]

**Pull request merged**